### ISSUE TITLE
Syntax error in example code

### ISSUE THREAD

**Issue by derekblank** [Globegitter/chartist-plugin-tooltip#14]
Looks like there's a syntax error in the example code:

```
var chart = new Chartist.Line('.ct-chart', {
  labels: [1, 2, 3],
  series: [
    [
      {meta: 'description', value: 1 }, 
      {meta: 'description', value: 5}, 
      {meta: 'description', value: 3}
    ],
    [
      {meta: 'other description', value: 2}, 
      {meta: 'other description', value: 4}, 
      {meta: 'other description', value: 2}
    ]
}, {
  plugins: [
    Chartist.plugins.tooltip()
  ]
});
```

The opening `[` bracket on line 3 never resolves. Seems like there should be a closing bracket after the series objects. I copied the example code verbatim to try as a proof of concept and received an error. May help others to fix it. Just sayin'.

**Comment by Globegitter** [comment]
PR to the readme is appreaciated :)

**Comment by ealves-pt** [comment]
Solved with PR #20.
